changelog/unreleased/kong/fix-core-pass-ctx-to-log-init-worker-errors.yml

```diff
@@ -0,0 +1,4 @@
+message: |
+   Fix to always pass `ngx.ctx` to `log_init_worker_errors` as otherwise it may runtime crash.
```

Comment:
I think this msg is very technical, could you make it more customer-friendly?

Reply:
@ADD-SP there is no way to make it customer friendly. Thus I was thinking about removing the whole changelog. But then I though people start complaining that it is not there.

Reply:
IMO it's perfectly fine to leave internal changes off the changelog.